app-office/planner/planner-0.14.6_p20130520-r3.ebuild

```diff
@@ -1,32 +1,35 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
 GNOME2_LA_PUNT="yes"
 
 inherit gnome2
 
 DESCRIPTION="Project manager for Gnome"
 HOMEPAGE="https://wiki.gnome.org/Apps/Planner"
-SRC_URI="https://dev.gentoo.org/~eva/distfiles/${PN}/${P}.tar.xz"
+if [[ "${PV}" == "9999" ]] ; then
+	EGIT_REPO_URI="https://gitlab.gnome.org/World/planner.git"
+	inherit git-r3
+	SRC_URI=""
+else
```

Comment:
This won't work, as git isn't autotools anymore but meson. So I'm not sure why you added this block compared to previous PR. The live ebuild in the other commit of course can have this, so it's ready to be copied over to the release version.

Reply:
@thesamesam asked for this. When you don't have that it one ebuild of either the live or none live ebuild one could assume its stale.